src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -78,10 +81,16 @@ manualRegistrationToolBox::manualRegistrationToolBox(QWidget *parent) : medRegis
     d->b_reset->setObjectName("resetButton");
     connect(d->b_reset,SIGNAL(clicked()),this,SLOT(reset()));
 
-    d->b_save = new QPushButton("Save",widget);
+    d->b_save = new QPushButton("Save Image",widget);
+    d->b_save->setToolTip(tr("Save registered image"));
     d->b_save->setObjectName("saveButton");
     connect(d->b_save,SIGNAL(clicked()),this,SLOT(save()));
 
+    d->b_saveTransformation = new QPushButton("Save Transformation",widget);
+    d->b_saveTransformation->setObjectName("Save Transformation");
```

Comment:
By the way, It would be better to name it "Export Transformation" i think, because "Save" is usually used for the database, and export is used when the user can choose the location.